tests/test_legacy.py

```diff
@@ -1,22 +1,283 @@
 import unittest
-
-import numpy as np
+from pydicom import FileDataset, Dataset
+from pydicom.uid import generate_uid
+from highdicom.legacy import sop
+from datetime import datetime, timedelta
 import pytest
+import enum
 
 
-class LegacyConvertedEnhancedMRImage(unittest.TestCase):
+class Modality(enum.IntEnum):
+    CT = 0
+    MR = 1
+    PET = 2
```

Comment:
Would it be more consistent to use PT here in place of PET?

Reply:
Though this would be more readable. Sure ...